collectors/apicmcpfault.py

```diff
@@ -44,10 +44,10 @@ def collect(self):
                     fault_summary = fetched_data['imdata'][x]['faultInst']['attributes']['dn']
                     fault_desc = fetched_data['imdata'][x]['faultInst']['attributes']['descr']
 
-                    LOG.debug("host: %s, ip: %s", fault_lifecyle, fault_summary, fault_desc)
+                    LOG.debug("host: %s", fault_lifecyle, fault_summary, fault_desc)
                     metric_counter += 1
 
-                    c_mcp_faults.add_metric(labels=[host, fault_lifecyle, fault_summary, fault_desc], value=1)
+                    c_mcp_faults.add_metric(labels=[host, fault_summary, fault_desc, fault_lifecyle], value=1)
```

Comment:
ordering of the labels must match with metric definition in line https://github.com/sapcc/apic-exporter/blob/7044892a3070c953b2f0282a0e9715784ef30074/collectors/apicmcpfault.py#L23

Reply:
Labels in add_metric (https://github.com/sapcc/apic-exporter/blob/master/collectors/apicmcpfault.py#L50) does not match with the labels in the metric definition above (line 23). Labels must be the same and exactly in the same order. If you don't have a value for the label **ip**, you should remove it from the metric definition (line 23.)